core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -132,13 +132,13 @@ void addChildContext(BaseEventContext childContext) {
   @Override
   public final void success() {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
+      if (debugLogEnabled) {
         LOGGER.debug(this + " empty response was already completed, ignoring.");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use the built-in formatting to construct this argument. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLOTwXotIXnmsUkh11&open=AWkLOTwXotIXnmsUkh11)

[ISSUE_KEY]: AWkLOTwXotIXnmsUkh11